packages/hts_shrink/hts_shrink.2.1.2/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/hts_shrink"
+bug-reports: "https://github.com/UnixJunkie/hts_shrink/issues"
+dev-repo: "git+https://github.com/UnixJunkie/hts_shrink.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {>="2.5"}
```

This needs to be there as your dune project indicates you depend on dune 2.5 or later.